wake/utils/transform.ts

```diff
@@ -303,6 +303,17 @@ export const toProduct = (
     });
   }
 
+  //@ts-expect-error kit prop exists on buy list
+  if (variant.kit) {
```

Comment:
Please don't ignore such type errors.

If you really need, set variant as a BuyList type (`variant: ProductFragment | SingleProductFragment | BuyList`), or work refactoring `toProduct` to deal with two distinct types.

Reply:
I will refact this functions in other PR